pkg/operator/ceph/controller/spec.go

```diff
@@ -106,11 +106,16 @@ PERIODICITY=%s
 LOG_ROTATE_CEPH_FILE=/etc/logrotate.d/ceph
 LOG_MAX_SIZE=%s
 ROTATE=%s
+ADDITIONAL_LOG_FILE=%s
 
 # edit the logrotate file to only rotate a specific daemon log
 # otherwise we will logrotate log files without reloading certain daemons
 # this might happen when multiple daemons run on the same machine
-sed -i "s|*.log|$CEPH_CLIENT_ID.log|" "$LOG_ROTATE_CEPH_FILE"
+if [ -n "$ADDITIONAL_LOG_FILE" ]; then
+    sed -i "s|*.log|$CEPH_CLIENT_ID.log $ADDITIONAL_LOG_FILE|" "$LOG_ROTATE_CEPH_FILE"
```

Comment:
@BlaineEXE if additional log file is not present we will run the original command `sed -i "s|*.log|$CEPH_CLIENT_ID.log|" "$LOG_ROTATE_CEPH_FILE"` since it's a if condition, am i missing anything here?

Reply:
As noted, I want to avoid the conditional branch. Can you test what happens if we merely keep line 115 and nix the rest, resulting in an extra space in the file when no additional files are used?

Reply:
For clarity, this is what I mean. I don't see a need for branching when I think this will do the same thing, assuming the extra space after `$CEPH_CLIENT_ID.log` isn't a problem when `ADDITIONAL_LOG_FILE=""`

```sh
# this might happen when multiple daemons run on the same machine
sed -i "s|*.log|$CEPH_CLIENT_ID.log $ADDITIONAL_LOG_FILE|" "$LOG_ROTATE_CEPH_FILE"

# replace default daily with given user input
sed --in-place "s/daily/$PERIODICITY/g" "$LOG_ROTATE_CEPH_FILE"
```